### DISCUSSION TITLE
modelscope vs zeroscope

### DISCUSSION THREAD

**Discussion by beepbeepplop** [deforum-art/sd-webui-text2video#209] [General]
Hi. 

Been using both recently and my feeling is zeroscope needs more training. There are just some videos that zeroscope cannot reproduce even from a previous modelscope video. Anyway, amazing work. Keep it up guys. I've been using pikalabs and they're not even close to the work you're all doing.

**Comment by cerspense** [comment]
Everything you just stated seems completely backwards. Do you have any examples with the same seed and prompts to back up your intuitions? Also for zeroscope are you using the recommended clip length and resolution for the model? And are you using the 576 model (XL is used best for vid2vid, not text2vid)

**Reply by beepbeepplop** [reply]
When I say it needs more training I'm saying it doesn't understand enough about the world. I have generated over 10K clips using both models. Generally speaking modelscope has given me what I want. For example, I wanted (for whatever reason) a clip of a baby going around in a washing machine. Modelscope couldn't do it until I provided it with something I keyframed in shotcut. Then I got it first go. I provided the same video to zeroscope and in hundreds of renders I have not once got the video I am after. It is not an issue of quality. Zeroscope is obviously providing higher quality videos. I am reporting my real world experience. As I say, this is an amazing project and it blows pikalabs efforts out of the water...
